egs/dihard_2018/v1/run.sh

```diff
@@ -28,9 +28,9 @@ stage=0
 if [ $stage -le 0 ]; then
   local/make_voxceleb2.pl $voxceleb2_root dev data/voxceleb2_train
   local/make_voxceleb2.pl $voxceleb2_root test data/voxceleb2_test
-  # This script creates data/voxceleb1_test and data/voxceleb1_train.
   # Our evaluation set is the test portion of VoxCeleb1.
```

Comment:
Can you remove the comment "Our evaluation set is the test portion of VoxCeleb1." It seems to be referring to a different recipe.